core_data_modules/traced_data/traced_data.py

```diff
@@ -317,6 +321,24 @@ def get_history(self, key):
         history.sort(key=lambda x: x["timestamp"])
         return history
 
+    def clear_history(self, user, file_sha):
```

Comment:
consider `flush_history` instead of clear?

Clear suggests that it deletes the history rather than archiving it onto disk

Reply:
Ignore - misunderstanding.